diagnostic.md

```diff
@@ -61,8 +61,8 @@ var result = strangeAdd(0, -5);
 ```
 
 What is the value of result?  Explain your answer.
+the value of the result is 
```

Comment:
The value of result is 5.  With 0 and -5 as arguments to strangeAdd(), the if-block's conditional expression evalutes to false so return a + b; doesn't run.  return a - b; runs and returns 0 - (-5) = 5.